models/core/txs.sql

```diff
@@ -1,170 +1,30 @@
---             "type": "STATICCALL",
---             "value": ""
---           }
---         ],
---         "from": "0x4f9450f17307c7ee7b0538ba622791a24f61d474",
---         "gas": "0xb9411",
---         "gasUsed": "0x4f6c",
---         "input": "0xd06ca61f00000000000000000000000000000000000000000000006bee7958323912d60000000000000000000000000000000000000000000000000000000000000000400000000000000000000000000000000000000000000000000000000000000005000000000000000000000000ef977d2f931c1978db5f6747666fa1eacb0d03390000000000000000000000000dc78c79b4eb080ead5c1d16559225a46b580694000000000000000000000000224e64ec1bdce3870a6a6c777edd450454068fec000000000000000000000000cf664087a5bb0237a0bad6742852ec6c8d69a27a0000000000000000000000006983d1e6def3690c4d616b13597a09e6193ea013",
---         "output": "0x",
---         "time": "",
---         "to": "0xf012702a5f0e54015362cbca26a26fc90aa832a3",
---         "type": "STATICCALL",
---         "value": ""
---       }
---     ],
---     "from": "0xe74be499e7020f0a2fd07ffdf4165a638c237c04",
---     "gas": "0xc4c66",
---     "gasUsed": "0xe16b",
---     "input": "0xe5546f37000000000000000000000000000000000000000000000000063eb89da4ed0000000000000000000000000000ef977d2f931c1978db5f6747666fa1eacb0d03390000000000000000000000006983d1e6def3690c4d616b13597a09e6193ea013000000000000000000000000c35dadb65012ec5796536bd9864ed8773abc74c40000000000000000000000001b02da8cb0d097eb8d57a175b88c7d8b47997506000000000000000000000000f012702a5f0e54015362cbca26a26fc90aa832a300000000000000000000000082fdff53547141d1a4439b0bfef34ffd366e77dc000000000000000000000000000000000000000000000000000000000000010000000000000000000000000000000000000000000000000000000000000001600000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000006000000000000000000000000000000000000000000000000000000000000000a00000000000000000000000000000000000000000000000000000000000000001000000000000000000000000e74be499e7020f0a2fd07ffdf4165a638c237c040000000000000000000000000000000000000000000000000000000000000005000000000000000000000000ef977d2f931c1978db5f6747666fa1eacb0d03390000000000000000000000000dc78c79b4eb080ead5c1d16559225a46b580694000000000000000000000000224e64ec1bdce3870a6a6c777edd450454068fec000000000000000000000000cf664087a5bb0237a0bad6742852ec6c8d69a27a0000000000000000000000006983d1e6def3690c4d616b13597a09e6193ea013",
---     "output": "0x0000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000000b616d6f6f756e74206e6f31000000000000000000000000000000000000000000",
---     "time": "748.855682ms",
---     "to": "0x4f9450f17307c7ee7b0538ba622791a24f61d474",
---     "type": "CALL",
---     "value": "0x0"
---   },
---   "transactionIndex": 0,
---   "v": "0xc6ac98a3",
---   "value": 0
--- }
\ No newline at end of file
+    tx :nonce :: string as nonce,
+    tx_block_index as "INDEX",
```

Comment:
totally understand why you're using the uppercase here but think snowflake is also accepting lower cases as all the columns in snowflake is auto uppercased :D